CompuCell3D/core/CompuCell3D/steppables/CMakeLists.txt

```diff
@@ -22,8 +22,8 @@ ADD_SUBDIRECTORY(PIFDumper)
 ADD_SUBDIRECTORY(BlobFieldInitializer)
 ADD_SUBDIRECTORY(UniformFieldInitializer)
 ADD_SUBDIRECTORY(PolygonFieldInitializer)
-# ADD_SUBDIRECTORY(PolygonFieldInitializer)
 ADD_SUBDIRECTORY(TubeFieldInitializer)
+ADD_SUBDIRECTORY(PolygonFieldInitializer)
```

Comment:
let's remove this one because it is identical to line 24 and cmake crashes when it sees duplicated

Reply:
Ah, a merge conflict mistake. Thank you for fixing it.